jetbrains-core/src/software/aws/toolkits/jetbrains/utils/CompatibilityUtils.kt

```diff
@@ -23,4 +40,59 @@ object CompatibilityUtils {
     @JvmStatic
     fun createPopupBuilder(validationInfo: ValidationInfo, configurator: Consumer<JEditorPane>?): ComponentPopupBuilder? =
         ComponentValidator::class.staticFunctions.find { it.name == "createPopupBuilder" }?.call(validationInfo, configurator) as? ComponentPopupBuilder
+
+    /**
+     * Can be removed when min-version is 19.1 FIX_WHEN_MIN_IS_191
+     */
+    @TestOnly
+    fun <T> registerExtension(name: ExtensionPointName<T>, extension: T, disposable: Disposable) {
+
+        val newRegister = ExtensionPoint::class.functions.find {
+            it.name == "registerExtension" &&
+                it.parameters.size == 2 &&
+                it.parameters[1].type == Disposable::class.createType()
+        }
+
+        if (newRegister != null) {
+            val ep = name.getPoint(null)
+            newRegister.call(ep, extension, disposable)
+        } else {
+            val legacyRegister = PlatformTestUtil::class.staticFunctions.find { it.name == "registerExtension" }
+            legacyRegister ?: throw NoSuchMethodError("Can't find PlatformTestUtil.registerExtension")
+            legacyRegister.call(name, extension, disposable)
+        }
+    }
+
+    /**
+     * Can be removed when min-version is 19.3 FIX_WHEN_MIN_IS_193
+     */
+    @TestOnly
+    fun createProject(path: Path): Project {
+        val heavyTestCaseClass = tryOrNull { Class.forName("com.intellij.testFramework.HeavyPlatformTestCase") }
+        if (heavyTestCaseClass != null) {
+            // New method
+            val method = heavyTestCaseClass.getMethod("createProject", Path::class.java)
+            return method.invoke(null, path) as Project
+        }
+        val legacyCreate = PlatformTestCase::class.java.getMethod("createProject", File::class.java, String::class.java)
+        return legacyCreate.invoke(null, path.toFile(), "Fake") as Project
+    }
+
+    /**
+     * FIX_WHEN_MIN_IS_192 WebModuleType.getInstance() is deprecated in 191, use WebModuleTypeBase#getInstance for version 191+
+     */
+    @TestOnly
```

Comment:
This is called from `NodeJsRuntimeGroup.kt` so it's definitely not `@TestOnly`

Reply:
yes - good catch